src/scenes/level_scene.py

```diff
@@ -843,7 +843,7 @@ def get_entity_on_tile(self, tile: Position) -> Optional[Entity]:
         return None
 
     def determine_path_to(
-        self, destination_tile: Position, distance_for_tile: dict[tuple[int, int], int]
+        self, destination_tile: Position, distance_for_tile: dict[tuple, int]
```

Comment:
Maybe it should be `Position` instead of tuple

Reply:
That makes sense, I think I made this change before I realized there was a Position class.

Reply:
Trying it again, and it may be because Position is a Union with pygame.Vector2, which isn't hashable for the set function.

Reply:
Indeed I remember something about it, can be annoying...